plutus-core/untyped-plutus-core/test/DeBruijn/Spec.hs

```diff
@@ -82,10 +82,15 @@ test_debruijn :: TestTree
 test_debruijn = runTestNestedIn ["untyped-plutus-core","test"] $
                   testNested "DeBruijn"
                     [testNested "Golden" $
-                      fmap (\ (n,t) -> nestedGoldenVsDoc n $ act t) tests
+                      fmap (\ (n,t) -> nestedGoldenVsDoc n $ actThrow t) tests
+                    ,testNested "GoldenGrace" $
```

Comment:
This is fine... but it reads a bit odd. I might just rename these two groups to "Strict" and "Graceful" or something. Conveys more information than "Golden" IMO.